src/app/common/loading-modal/loading-snackbar.service.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { TestBed } from '@angular/core/testing';
+
+import { LoadingSnackbarService } from './loading-snackbar.service';
+import { MaterialModule } from '../../material.module';
+
+describe('LoadingDialogService', () => {
+  let service: LoadingSnackbarService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [MaterialModule],
+      providers: [
+        LoadingSnackbarService,
+      ],
+    });
+    service = TestBed.inject(LoadingSnackbarService);
+  });
+
+  it('should be created', () => {
```

Comment:
Instead of this you should test `show` and `hide` methods - `MaterialModule` should not be imported as this is supposed to be unit test. Test especially if error is thrown if `hide` is called before `show`